tools/Makefile

```diff
@@ -24,6 +24,7 @@ venv:
 
 espnet.done: venv
 	. venv/bin/activate; pip install pip --upgrade
+	. venv/bin/activate; pip install numpy
```

Comment:
I think it's better to ask people developing nara_wpe to include numpy installation.
It's OK for now, but please add comments to remove this line in future after nara_wpe includes the numpy installation.

Reply:
added comment.